packages/integration-tests/lambda.js

```diff
@@ -2,10 +2,26 @@
 
 const { lambda } = require('@cumulus/aws-client/services');
 
+/**
+ * Retrieve a rule's Kinesis Event Source Mappings
+ *
+ * @param {Object} uuid - unique identifier for a rule
+ * @returns {Promise<Object>} - details about an Event Source Mapping
+ */
 async function getEventSourceMapping(uuid) {
   return await lambda().getEventSourceMapping({ UUID: uuid }).promise();
 }
 
+/**
+ * Delete a rule's Kinesis Event Source Mappings
+ *
+ * @param {Object} uuid - unique identifier for a rule
+ * @returns {Promise<void>}
```

Comment:
Is this actually returning `void`?

Reply:
That's what the return of `deleteEventSourceMapping` claims but we're `promise()`ing it so I suppose it should either match the AWS return type:

`Promise<PromiseResult<AWS.Lambda.EventSourceMappingConfiguration, AWS.AWSError>>`

or `unknown` as above because we don't have all of those types imported? What do you think?

Reply:
Updated here but open to discussion: https://github.com/nasa/cumulus/pull/3149/commits/4df26b664bfa0933ca41ddc604bf6a12222af1de